### ISSUE TITLE
ksonnet.io website not accessible

### ISSUE THREAD

**Issue by sorbra** [ksonnet/ksonnet#694]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
ksonnet.io website does not respond. Makes it hard to follow tutorials etc.

**What you expected to happen**:
To be able to open the website :)

**How to reproduce it (as minimally and precisely as possible)**:
Try to open http://ksonnet.io i a web browser

**Anything else we need to know?**:
I am located in Denmark. I have tried to connect via various internet providers, and the issue does not appear to be specific to my internet connection.

**Environment**:
- ksonnet version (use `ks version`): N/A, but 0.11.0
- Kubernetes version (use `kubectl version`): N/A, but minikube v0.28.0

**Comment by bryanl** [comment]
This should be resolved
